metadata.json

```diff
@@ -27,6 +27,10 @@
     {
       "name": "herculesteam/augeasproviders_core",
       "version_requirement": ">= 2.1.0 < 3.0.0"
+    },
+    {
+      "name": "camptocamp/systemd",
```

Comment:
Given this is only needed on systems that run systemd and Puppet < 6.1.0, should we make this a soft dependency documented in README?

Reply:
Removed dependency and added a section to the README.